### DISCUSSION TITLE
stock avail but SM said out of stock

### DISCUSSION THREAD

**Discussion by l0g4n1337** [jef/streetmerchant#3112] [Help]
Ok so i have added nvidia-it store and the 4080 butt something is wrong ... look at this, hope someone can help me to fix!

![firefox_opTOpx64cI](https://user-images.githubusercontent.com/49865595/207590759-aca90da0-190d-4893-89f7-cb8d1ef3aea4.png)

and anyways after 1 min i got error 403 

![firefox_r9MCz2R4fB](https://user-images.githubusercontent.com/49865595/207591228-120cf5f6-9387-4e2d-ac4e-f22cfa4a1ecd.png)

**Comment by Mr-Tech-13** [comment]
A 403 is normal for Nvidia, use a vpn/proxy to get around this (temporarily). Make sure that you have the right selector on the page

**Comment by crixeva** [comment]
这是来自QQ邮箱的自动回复邮件。您好，您的邮件我已经收到了，我会尽快给您回复。

**Reply by Mr-Tech-13** [reply]
What do you mean by automated email from QQ mail? Also how did you get an email, unless you subscribed to notifications
